README.md

```diff
@@ -244,3 +244,45 @@ database:
       connection-timeout: 30
 ```
 
+
+### Database migration
+
+Initialize and migrate database using provided and configured `io.github.balconyseats.vertx.helper.database.migration.DatabaseMigration`.  
+Currently is implemented Flyway database migration with `io.github.balconyseats.vertx.helper.database.migration.ConfigFlywayDatabaseMigration`.
+
+To use database migration create `io.github.balconyseats.vertx.helper.database.migration.ConfigDatabaseMigrationInitializationHandler`  
+and add it to `VertxApplication` as pre or post handler.
+
+```java
+
+VertxApplication vertxApplication = VertxApplication.builder()
+    ...
+    .preHandler(InitializationHandler.composite(
+            ConfigDatabaseMigrationInitializationHandler.instance()
+        )
+    ) // add here 
+    ...
+    .postHandler(InitializationHandler.composite(
+            ConfigDatabaseMigrationInitializationHandler.instance()
+        )
+    ) // or here
+    .build();
+
+```
+
+Configuration uses database as parent and use properties from database such as user, password, host, etc..:
```

Comment:
`It uses user, password, host, etc. properties from database configuration properties:`